tests/e2e/standalone/utils.go

```diff
@@ -142,7 +142,7 @@ func ensureDaprInstallation(t *testing.T) {
 	// Slim mode does not have any components by default.
 	// Install the components required by the tests.
 	if isSlimMode() {
-		err = createSlimComponents(filepath.Join(daprPath, "components"))
+		err = createSlimComponents(filepath.Join(daprPath, "resources"))
```

Comment:
use the const declared in common.go instead of string literal for "resources"

Reply:
+1, I would also suggest using typed values instead of string in CLI (that needs some refactoring too). But for example, imageVariant, path, etc., everything is a string today. Method signatures like these can get confusing.
```go
func method(var1, var2, var3 string)
```
Instead this is better
```go
func method(var1 t1, var2 t2, var3 t3)
```

Reply:
changing this to be something like- 
```
type DirName string

const (
DEFAULTCOMPONENTSDIRNAME DirName = "components"
DEFAULTRESOURCESDIRNAME  DirName = "resources"
)
```

Reply:
Or maybe `DaprDir`, which makes the intent further clear.

```go
// DaprDir is a directory in the Dapr CLI installation path.
type DaprDir string

const (
BIN_DAPR_DIR DaprDir = "bin"
COMPONENTS_DAPR_DIR DaprDir = "components"
RESOURCES_DAPR_DIR  DaprDir = "resources"
)
```
@mukundansundar thoughts?

Reply:
Using the const from the commons.go gives error `panic: proto: file "dapr/proto/common/v1/common.proto" is already registered`

Reply:
should we add `export GOLANG_PROTOBUF_REGISTRATION_CONFLICT=warn/ignore` for workflows ?

Reply:
moved the varibales to utils.go